src/components/list/list.tsx

```diff
@@ -28,11 +33,26 @@ const defaultProps = {
   mode: 'default',
 }
 
-export const List: FC<ListProps> = p => {
+export type ListRef = {
+  nativeElement: HTMLDivElement | null
+}
+
+export const List = forwardRef<ListRef, ListProps>((p, ref) => {
   const props = mergeProps(defaultProps, p)
+  const nativeListRef = useRef<HTMLDivElement>(null)
```

Comment:
这里感觉叫 `nativeElementRef` 更合适一些……

```suggestion
  const nativeElementRef = useRef<HTMLDivElement>(null)
```